### PR TITLE
Issue 5428 for SchemaChangedListenerTest.java

### LINKED ISSUE #5428: Use YAML file instead of inline string in test cases
Some cases include YAML inline string for testing orchestration persistent which are hard to read.
It is better to put the YAML inline string into YAML file.

Some test cases which need to be refactored:

- [ ] ConfigCenterTest
- [x] SchemaChangedListenerTest
- [ ] OrchestrationSpringBootRegistryShardingTest
- [ ] OrchestrationSpringBootRegistryEncryptTest
- [ ] OrchestrationSpringBootRegistryMasterSlaveTest
- [ ] HttpServerHandlerTest
- [ ] Still to be found...

### PR DESCRIPTION
**Pull request by hitnewbee**
For #5428 .

Changes proposed in this pull request:
- Move inline string to yaml file in resource folder
- Add method to read yaml as string

### CODE REVIEW

File: sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-configcenter/src/test/java/org/apache/shardingsphere/orchestration/core/configcenter/listener/SchemaChangedListenerTest.java [hunk 1]
```diff
@@ -120,16 +83,33 @@ public void assertCreateIgnoredEvent() {
...
         assertThat(((DataSourceChangedEvent) actual).getShardingSchemaName(), is("sharding_db"));
     }
+
+    private String readYamlIntoString(final String yamlFile) throws IOException {
+        File file = new File(Preconditions.checkNotNull(getClass().getClassLoader().getResource(yamlFile), "file resource `%s` must not be null.", yamlFile).getFile());
+        StringBuilder yamlContent = new StringBuilder();
+        FileReader fileReader = new FileReader(file);
+        BufferedReader reader = new BufferedReader(fileReader);
+        String line;
+        while (null != (line = reader.readLine())) {
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-configcenter/src/test/java/org/apache/shardingsphere/orchestration/core/configcenter/listener/SchemaChangedListenerTest.java:100]: Why need to skip comment here? Can snake yaml process the comment?

File: sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-configcenter/src/test/resources/sharding-rule.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    t_order:
+      logicTable: t_order
+      actualDataNodes: ds_${0..1}.t_order_${0..1}
+      tableStrategy:
+        standard:
+          shardingAlgorithm:
+            type: INLINE
+            props:
+              algorithm.expression: t_order_${order_id % 2}
+          shardingColumn: order_id
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-configcenter/src/test/resources/sharding-rule.yaml:30]: Please leave a new line in the end of file

File: sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-configcenter/src/test/java/org/apache/shardingsphere/orchestration/core/configcenter/listener/SchemaChangedListenerTest.java (hunk 1 above)
Comment (hitnewbee) [reply sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-configcenter/src/test/java/org/apache/shardingsphere/orchestration/core/configcenter/listener/SchemaChangedListenerTest.java:100]: OK, I will try snake yaml

**Comment by kimmking** [comment]
conflict should be solved.

**Pull request merged**